fiftyone/operators/executor.py

```diff
@@ -889,6 +889,11 @@ def store(self, store_name):
         dataset_id = self.dataset._doc.id
         return ExecutionStore.create(store_name, dataset_id)
 
+    def _get_serialized_request_params(self):
```

Comment:
Not really liking this as the interface...but didn't feel that changing `.serialize()` was the right move. However there is no need for `panel_state` in a serialized `ctx`. Instead it should really be outside of `request_params` to begin with. I've started that change elsewhere, but its a much more complex change and regresses some functionality in the dashboard plugin.